portfolio/src/main/webapp/script.js

```diff
@@ -16,14 +16,58 @@ function getComments() {
   fetch('/data').then(response => response.json()).then((commentsArray) => {
     let container = document.getElementById('comments-container');
     container.innerText = '';
+    console.log(commentsArray);
     for (const comment of commentsArray) {
-        container.appendChild(createListElement(comment));
+        container.append(createCommentView(comment));
     }
   });
 }
 
+function initMap() {
+  const prefPlace = { lat: 55.17477877, lng: 61.29954625 };
+  const map = new google.maps.Map(document.getElementById("map"), {
+    center: prefPlace,
+    zoom: 12,
+    mapTypeId: 'hybrid'
+  });
+  marker = new google.maps.Marker({position: prefPlace, map: map});
+}
+
+function loadAutorizationView() {
+  fetch('/autorization').then(response => response.text()).then(data => {
+    view = document.getElementById('autorization_view')
+    view.insertAdjacentHTML("afterbegin", data);
+  });
+}
+
+function processCommentViewStatus() {
+  fetch('/login_status').then(response => response.text()).then(status => {
+    commentView = document.getElementById('comment-form')
+    console.log("CommentView is: " + commentView)
+    if (status === "true") {
+      commentView.style.display = "block";
+    } else {
+      commentView.style.display = "none";
+    }
+  });
+}
+
+function doPreparation() {
+  processCommentViewStatus();
+  getComments();
+  loadAutorizationView();
+}
+
+function createCommentView(comment) {
+  div = document.createElement("div");
+  div.className = "comment-view";
+  div.innerHTML = "<p class = \"email\"> " + comment.userEmail + "</p>\
+          <p class = \"comment-message\">" +  comment.message + "</p>";
```

Comment:
What happens if a malicious user submits a message that says:

```javascript
  <script>alert("you've been pwned!")</alert>
```

This code will inject that script straight into the page and run it. We probably want to prevent scripting attacks like this. One possible fix would be to use the direct DOM manipulation rather than text:

```javascript
  const email = document.createElement("p");
  email.className = "email";
  email.innerText = comment.userEmail;  // note: innerText, not innerHTML.
  const message = document.createElement("message");
  message.className = "comment-message";
  message.innerText = comment.message;  // note: innerText, not innerHTML.
  document.appendChild(email);
  document.appendChild(message);
```